crates/bevy_gizmos/src/gizmos.rs

```diff
@@ -459,6 +460,45 @@ impl<'s> Gizmos<'s> {
         }
     }
 
+    /// Draw an arc.
+    ///
+    /// # Example
+    /// ```
+    /// # use bevy_gizmos::prelude::*;
+    /// # use bevy_render::prelude::*;
+    /// # use bevy_math::prelude::*;
+    /// # use std::f32::consts::PI;
+    /// fn system(mut gizmos: Gizmos) {
+    ///     gizmos.arc_2d(Vec2::ZERO, 0., PI / 4., 1., Color::GREEN);
+    ///
+    ///     // Arcs have 32 line-segments by default.
+    ///     // You may want to increase this for larger arcs.
+    ///     gizmos
+    ///         .arc_2d(Vec2::ZERO, 0., PI / 4., 5., Color::RED)
+    ///         .segments(64);
+    /// }
+    /// # bevy_ecs::system::assert_is_system(system);
+    /// ```
+    #[inline]
+    pub fn arc_2d(
+        &mut self,
+        position: Vec2,
+        center_angle: f32,
```

Comment:
The center_angle vs arc_angle needs to be clearly differentiated and explained in the docs.

Reply:
I hesitated between centre_angle and direction_angle. What do you think is better? I agree that this should be explained no matter what attribute name we choose though. I'll work on that.

Reply:
I think direction_angle is much clearer.